march_hardware/include/march_hardware/error/error_type.h

```diff
@@ -31,6 +31,7 @@ enum class ErrorType
   MISSING_REQUIRED_KEY = 119,
   INIT_URDF_FAILED = 120,
   INVALID_SW_STRING = 121,
+  SLAVE_LOST_TIMOUT = 122,
```

Comment:
Could you also add an error description in project-march/tutorials, if this is merged? :innocent:

Reply:
yes!